### PR TITLE
Add a link to the bind method on the sql() function page

### PR DESCRIPTION
**Pull request by Eijebong**
This improves discoverability of bound values.

[ci skip]

### CODE REVIEW

**Comment by killercup** [comment]
More links are more good.

File: diesel/src/expression/sql_literal.rs [hunk 1]
```diff
@@ -139,6 +139,12 @@ impl<ST> NonAggregate for SqlLiteral<ST> {
 /// DSL. You will need to provide the SQL type of the expression, in addition to
 /// the SQL.
 ///
+/// # Bound parameters
+///
+/// If you need to pass arguments to your query, you should use [`.bind()`].
+///
+/// [`.bind()`]: struct.SqlLiteral.html#method.bind
```
Comment (killercup) [inline diesel/src/expression/sql_literal.rs]: Does this actually work? This is rendered on the `/diesel/expression/dsl/fn.sql.html` page, right? And `bind` is on `/diesel/expression/sql_literal/struct.SqlLiteral.html#method.bind`

Comment (Eijebong) [inline diesel/src/expression/sql_literal.rs]: No it doesn't... I think I didn't wait for `cargo doc` to finish before testing.

Comment (killercup) [inline diesel/src/expression/sql_literal.rs]: Someone should write an RFC for automatic linking in rustdoc! :trollface:

**Pull request merged**